libs/community/tests/integration_tests/vectorstores/test_vdms.py

```diff
@@ -36,19 +38,19 @@ def vdms_client() -> vdms.vdms:
 
 
 @pytest.mark.requires("vdms")
+@pytest.mark.enable_socket
```

Comment:
Why is this needed?

Reply:
VDMS client uses sockets so without it all test that use our client fail:
![image](https://github.com/langchain-ai/langchain/assets/33070862/b37fa48a-280e-414b-a5fc-0b559c6c7e8e)

Reply:
sockets are only blocked for unit tests not integration tests. How are you launching the test?